components/logicbutton/index.css

```diff
@@ -40,6 +40,7 @@
 	padding: var(--mod-logic-button-padding, var(--spectrum-component-edge-to-text-50));
 
 	border-width: var(--mod-logic-button-border-width, var(--spectrum-border-width-200));
+	border-style: solid;
```

Comment:
I noticed the weird border was occurring on logic button, similar to the border we saw on action button. 
<img width="280" alt="Screenshot 2025-05-02 at 12 31 45 PM" src="https://github.com/user-attachments/assets/bb3256b2-5c28-425f-97ed-5d9fe0a5fdef" />
<img width="277" alt="Screenshot 2025-05-02 at 12 31 36 PM" src="https://github.com/user-attachments/assets/4d51aa1d-624c-472a-9c74-74a234bb54f8" />